packages/types/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@trustwallet/types",
+  "version": "0.0.12",
+  "description": "Trust common types",
+  "homepage": "https://github.com/trustwallet/web-core#readme",
+  "keywords": [],
+  "author": "TrustWallet",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/trustwallet/web-core/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/trustwallet/web-core.git"
+  },
+  "main": "",
+  "types": "index.d.ts",
+  "scripts": {},
+  "dependencies": {
+    "@types/mocha": "^5.2.6"
```

Comment:
Why is this a dependency for this types package?

Reply:
Removed!